crev-lib/src/local.rs

```diff
@@ -88,6 +159,74 @@ fn https_to_git_url_test() {
     );
 }
 
+fn parse_git_repository_name(git_https_url: &str) -> Option<String> {
+    let parts = git_https_url.split('/');
```

Comment:
This could just call `parse_git_url_https` instead, no? No need to have that logic multiple times.

Reply:
X